HealthMonitoring.Forwarders.Influxdb/HealthMonitoring.Forwarders.Influxdb.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
If this is only for testing then I would rename HealthMonitoring.Forwarders.Influxdb to something showing that it is purely for tests ;-)